문자열 다루기 기본.md

```diff
@@ -0,0 +1,13 @@
+func solution(s string) bool {
+    if len(s) != 4 && len(s) != 6 {
+        return false
+    }
+    
+    for _, char := range s {
+        if !unicode.IsDigit(char) {
+            return false
+        }
+    }
```

Comment:
<img width="488" alt="스크린샷 2023-03-04 오후 6 58 50" src="https://user-images.githubusercontent.com/4418255/222893647-83f139c9-4e42-47de-b47d-92ac54d47db4.png">


unicode  is undefined 🥲